ignite/services/chain/chain.go

```diff
@@ -442,10 +429,16 @@ func (c *Chain) Commands(ctx context.Context) (chaincmdrunner.Runner, error) {
 		return chaincmdrunner.Runner{}, err
 	}
 
-	validator := config.Validators[0]
-	servers, err := validator.GetServers()
-	if err != nil {
-		return chaincmdrunner.Runner{}, err
+	var servers chainconfigv1.Servers
+
+	if len(config.Validators) == 0 {
+		servers = chainconfigv1.DefaultServers()
+	} else {
+		validator := config.Validators[0]
+		servers, err = validator.GetServers()
+		if err != nil {
+			return chaincmdrunner.Runner{}, err
+		}
 	}
```

Comment:
```suggestion
	servers := chainconfigv1.DefaultServers()

	if len(config.Validators) > 0 {
		validator := config.Validators[0]
		servers, err = validator.GetServers()
		if err != nil {
			return chaincmdrunner.Runner{}, err
		}
	}
```

Up to you but I found it more readable like that.